scene/resources/curve.cpp

```diff
@@ -1542,6 +1542,19 @@ Vector3 Curve3D::_calculate_tangent(const Vector3 &p_begin, const Vector3 &p_con
 	return p_begin.bezier_derivative(p_control_1, p_control_2, p_end, p_t).normalized();
 }
 
+// Static function to avoid code repetition. Internal to this file.
+static inline Vector3 _ensure_valid_up_vector(const Vector3& p_forward, const Vector3& p_up_one, const Vector3& p_up_another) {
```

Comment:
```suggestion
static inline Vector3 _ensure_valid_up_vector(const Vector3 &p_forward, const Vector3 &p_up_one, const Vector3 &p_up_another) {
```
The position matters

Reply:
right, thanks. fixed.